src/rebootstrap-raft/main.go

```diff
@@ -49,7 +50,7 @@ type rebootstrapCommand struct {
 	dryRun    bool
 	raftDir   string
 	apiPort   int
-	machineID string
+	machineID int
```

Comment:
Machine IDs are often represented as strings in Juju, never really known why, but this makes it kinda inconsistent from the other code.

Reply:
I wanted to add a sanity check on the ID to make sure it's actually a number which introduces a bunch of back and forth transformations if the type is `string`.

Is the machine ID ever not an `int`?

Reply:
Thanks. I changed the type back to string.

Reply:
The content is always an integer, but to be consistent with other named things, we have it as a string.